### PR TITLE
Add co2 intensity tooltips

### LINKED ISSUE #283: Country panel tooltip should show emissions when "emissions" are selected
When "emissions" are selected, country panel tooltip should show emissions instead of power.

image

### PR DESCRIPTION
**Pull request by corradio**
Fixes #283 and #455 

cc @brunolajoie @ThierryOllivero 

Checklist:
- [x] Update all translations
- [x] Better toggle (fixes https://github.com/tmrowco/electricitymap/issues/335)

image

### CODE REVIEW

File: web/app/configs/lang.json [hunk 1]
```diff
@@ -141,9 +141,17 @@
         "importfrom": "of import from",
         "productionimport": "production/import",
         "consumption": "consumption",
-        "electricityfrom": "electricity comes from",
-        "electricityto": "electricity is exported to",
-        "electricitystored": "electricity is stored using",
+
+        "electricityComesFrom": "{1} % of electricity in <img id=\"country-flag\"></img> <b>{2}</b> comes from {3}",
```
Comment (corradio) [inline web/app/configs/lang.json]: @Raffox97 what do you think about this sentence structure for translations? It will be much easier!

**Comment by Raffox97** [comment]
Yes, it will be so easy with that 👍

Il 3 mag 2017 11:57, "Olivier Corradi" <notifications@github.com> ha
scritto:

File: web/app/configs/lang.json (hunk 1 above)
Comment (magol) [inline web/app/configs/lang.json]: Big improvement. But i don't like the html in the string, it become garderob do translate and review. Please use a placeholder instead.

Comment (corradio) [inline web/app/configs/lang.json]: What do you mean by placeholder?

Comment (magol) [inline web/app/configs/lang.json]: It is enough to have `"{1} % of electricity in {2} comes from {3}"`
`{2}` is then replaced by `</img> <b>{2}</b>`

Or as a alternative use: `"{1} % of electricity in {2} {3} comes from {4}"`
where {2} is replaced by `</img>` 
and {3} is replaced by `<b>{2}</b>`

In that way it is easier to translate and easier to change the html without to having to change each translation.

**Comment by corradio** [comment]
Makes sense!

**Comment by brunolajoie** [comment]
Hello @oli, I have noticed figures beyond 100% in the new tooltip (see picture attached). Shouldn't we calculate the % as for electricity, saying we divide carbon emissions from source-x divided by (carbon emissions produced inland + imported)?
image

**Comment by corradio** [comment]
fixed

**Pull request merged**